samples/msal-node-samples/auth-code-cli-brokered-app/README.md

```diff
@@ -34,7 +34,7 @@ const config = {
 
 On the Authenticaion Page of your app, add the following:
 
-- In the **Redirect URI** section, select **Public client/native (mobile & desktop)** in the combo-box and enter the following redirect URI: `ms-appx-web://Microsoft.AAD.BrokerPlugin/<your-client-id>`, replacing `<your-client-id>` with the **Application (client) ID** from your app's registration screen.
+- In the **Redirect URI** section, select **Public client/native (mobile & desktop)** in the combo-box and enter the following redirect URI: `ms-appx-web://Microsoft.aad.BrokerPlugin/<your-client-id>`, replacing `<your-client-id>` with the **Application (client) ID** from your app's registration screen.
```

Comment:
I'm not sure if redirect URIs are case sensitive but is this change necessary?

Reply:
It is not. Updated script will prevent this from happening.  Thank you!